machine_configuration/machine-config-node-disruption.adoc

```diff
@@ -0,0 +1,41 @@
+:_content-type: ASSEMBLY
+:context: machine-configs-configure
+[id="machine-config-node-disruption_{context}"]
+= Using node disruption policies to minimize disruption from machine config changes
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+By default, when you make certain changes to the fields in a `MachineConfig` object, the Machine Config Operator (MCO) drains and reboots the nodes associated with that machine config. However, you can create a _node disruption policy_ that defines a set of changes to some Ignition config objects that would require little or no disruption to your workloads.
+
+A node disruption policy allows you to define the configuration changes that cause a disruption to your cluster, and which changes do not. This allows you to reduce node downtime when making small machine configuration changes in your cluster. To configure the policy, you modify the `MachineConfiguration` object, which is in the `openshift-machine-config-operator` namespace. See the example node disruption policies in the `MachineConfiguration` objects that follow.
+
+[NOTE]
+====
+There are machine configuration changes that always require a reboot, regardless of any node disruption policies. For more information, see _About the Machine Config Operator_.
+====
+
+After you create the node disruption policy, the MCO validates the policy to search for potential issues in the file, such as problems with formatting. The MCO then merges the policy with the cluster defaults and populates the `status.nodeDisruptionPolicyStatus` fields in the machine config with the actions to be performed upon future changes to the machine config. The configurations in your policy always overwrite the cluster defaults.
```

Comment:
nit: the MCO doesn't really validate the policy. The configuration will be interactively rejected at the API level instead if you apply a bad configuration. Maybe clarify so the user doesn't look for errors in the MCO